pkg/check/load/load.go

```diff
@@ -270,6 +299,39 @@ func (c *LoadCheck) checkCommittedDepth(ctx context.Context, client *bee.Client,
 	}
 }
 
+type test struct {
```

Comment:
This is also already existing in smoke, and we can expose it there and use it in load.